requirements/common.txt

```diff
@@ -26,11 +27,15 @@ MarkupSafe==0.23
 mccabe==0.4.0
 pep8==1.7.0
 psycopg2==2.6.1
+pyaml==16.11.4
 pyflakes==1.0.0
 pytz==2016.3
+PyYAML==3.12
 ratelim==0.1.6
+redis==2.10.5
 requests==2.9.1
 requirements==0.1
+rq==0.5.6
```

Comment:
It's used in flask-base, so we'll need it eventually. I can remove it for now though if you want.